restygwt/src/test/java/org/fusesource/restygwt/server/complex/ParameterizedTypeServlet.java

```diff
@@ -66,4 +75,18 @@ protected Integer getThing()
 			return thing;
 		}
 	}
+	
+	
+	public static class EchoNameServlet extends HttpServlet
+	{
+		@Override
+		protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException
```

Comment:
Without the fix in this JIRA issue, the flow of events would be

1. Get a reader for the interface type 
2. Jackson would find the `@JsonTypeInfo` annotation
3. Jackson would look for the 'type' property that indicates the implementation type, but it would be null due to changes in PR #272
4. The type ID resolver would receive the null and fail on a `Class.forName` call